state/cockroachdb/metadata.yaml

```diff
@@ -21,9 +21,9 @@ authenticationProfiles:
       - name: connectionString
         required: true
         sensitive: true
-        description: The connection string for the PostgreSQL database
+        description: The connection string for the Cockroach database
```

Comment:
The name is CockroachDB, which is a trademark, so we should respect that: https://en.wikipedia.org/wiki/CockroachDB